### ISSUE TITLE
Bug: nearly every time when opening a patch the file dialog returns to the plugdata folder

### ISSUE THREAD

**Issue by dromer** [plugdata-team/plugdata#1926]
This is very annoying. Sometimes it will show the directory of the currently opened patch, but 9 out of 10 times it will revert back to the plugdata documents folder.

This causes a lot of navigation issues when you are trying to open patches from a specific folder and you have to click a dozen times to get back to where you want to be.

Expected: file browser always starts in the same folder as the currently opened patch/tab.

**Comment by timothyschoen** [comment]
> Expected: file browser always starts in the same folder as the currently opened patch/tab.

I think the normal behaviour is that it would open the folder you last opened using the file browser?

**Comment by dromer** [comment]
Yeah I guess. This would work OK too, but that's certainly _not_ the behavior I'm getting.

Most of the time it reverts back to the plugdata folder which I've almost never needed to be in.

**Comment by timothyschoen** [comment]
be0142ac4 fixes at least one bug related to this, but there may be more bugs left.

**Comment by timothyschoen** [comment]
Should be fixed!

**Comment by dromer** [comment]
Seems to be !
